ddtrace/contrib/flask/__init__.py

```diff
@@ -27,6 +27,7 @@ def index():
 
     ddtrace-run python app.py
 
+Note: ddtrace does not support runpy. Avoid using ``python -m`` to run your flask app.
```

Comment:
do we want this note, or can we fix the issue?

Reply:
We can actually close out this change. This PR should introduce runpy support as a side effect: https://github.com/DataDog/dd-trace-py/pull/3358